README.md

```diff
@@ -26,52 +25,20 @@ Since anything in our codebase can be extended, overwritten, or installed as a p
 
 # Getting started
 
-### Requirements
-
-Reaction requires Meteor, Git, Mongo DB, OS-specific build tools. For step-by-step instructions, follow the documentation for [OS X](https://docs.reactioncommerce.com/docs/next/installation-osx), [Windows](https://docs.reactioncommerce.com/docs/next/installation-windows) or [Linux](https://docs.reactioncommerce.com/docs/next/installation-linux).
-
-### Install and run with CLI
-
-Install the [Reaction CLI](https://github.com/reactioncommerce/reaction-cli) to get started with Reaction:
-
-```sh
-npm install -g reaction-cli
-```
-
-Create your store:
-
-```sh
-reaction init
-cd reaction
-reaction
-```
-
-Open `localhost:3000`
-
-Learn more on how to [configure your project](https://docs.reactioncommerce.com/reaction-docs/master/configuration).
-
-Having installation issues? Check out our [troubleshooting docs](https://docs.reactioncommerce.com/docs/next/troubleshooting-development).
-
-### Install and run with Docker
-
-You can also run the app locally using [`docker-compose`](https://docs.docker.com/compose/) by running:
-
-```sh
-docker network create api.reaction.localhost
-docker-compose up
-```
+## Reaction 2.0
 
-Open `localhost:3000`
+Follow the documentation for [Reaction Platform](https://docs.reactioncommerce.com/docs/installation-reaction-platform) for all operating systems.
 
-This will use the `docker-compose.yml` file.
+## Pre-2.0 Reaction
 
-To learn more on how to develop on Docker, read our documentation on [developing Reaction on Docker](https://docs.reactioncommerce.com/docs/next/installation-docker-development) and [troubleshooting Docker](https://docs.reactioncommerce.com/docs/next/troubleshooting-development#docker-issues).
+Follow the documentation for installing pre-2.0 Reaction on [OS X](https://docs.reactioncommerce.com/docs/next/installation-osx), [Windows](https://docs.reactioncommerce.com/docs/next/installation-windows) or [Linux](https://docs.reactioncommerce.com/docs/next/installation-linux).
```

Comment:
@ticean @spencern I wasn't sure whether to entirely remove the pre-2.0 instructions or not... so I made a section like this.

Should I remove this section?

Reply:
I'm not opposed to keeping it somewhere - would it make sense to link to a pre-2.0 version of the docs as well then instead of `next`?

Reply:
Maybe something like "Looking for docs to install pre-2.0 Reaction?"